lib/phoenix/live_dashboard/helpers/live_helpers.ex

```diff
@@ -26,6 +26,7 @@ defmodule Phoenix.LiveDashboard.LiveHelpers do
         action: socket.assigns.live_action,
         node: found_node || node(),
         metrics: session["metrics"],
+        os_mon: session["os_mon"],
```

Comment:
We don't need this I believe. We can detect what is running directly from os_mon, no?

Reply:
To be clear, what I mean is that we can call `get_application` directly on the LiveView. The data doesn't have to come from the live_dashboard route. :)